torch/lib/c10d/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake ${CMAKE_MODULE_PATH})
+
+add_library(store Store.cpp FileStore.cpp)
+target_compile_options(store PUBLIC "-std=c++11")
```

Comment:
This cmake isn't going to be sufficient for install c10d, but I suppose we can fix it later

Reply:
I want to get something going first, fine if it is isolated and untested for a while.